components/pages/people/relationships/add/AddRelationshipForm.tsx

```diff
@@ -31,11 +32,20 @@ const AddRelationshipForm: React.FC<{
     return data;
   };
 
-  const { data: relationships, error } = useRelationships(personId);
+  const { data: secondPerson, error: secondPersonError } =
+    useResident(secondPersonId);
 
-  if (!relationships && !error) return <Spinner />;
+  const { data: relationships, error: relationshipsError } =
+    useRelationships(personId);
 
-  if (error || relationships === undefined) return <ErrorMessage />;
+  if (!secondPerson && !secondPersonError) return <Spinner />;
+
+  if (secondPersonError || secondPerson === undefined) return <ErrorMessage />;
```

Comment:
is there an error message we can show to the user here? `<ErrorMessage/>` can take a string as an argument

Reply:
Ah okay, we can update these to have more specific error messages for each hook.